public/static/locales/fr/donate.json

```diff
@@ -62,7 +63,7 @@
   "titleToShare": "Planter des arbres contre la crise climatique !",
   "giftToName": "Don à",
   "plantTreesWith": "Planter des arbres avec",
-  "textToShare": "Rejoignez {{name}} et des milliers d'autres pour planter un billion d'arbres pour lutter contre la crise climatique ! #StopTalkingStartPlanting #TrillionTrees @trilliontrees",
+  "textToShare": "Rejoignez {{name}} et des milliers d'autres pour planter un billion d'arbres pour lutter contre la crise climatique ! #StopTalkingStartPlanting #TrillionTrees@trilliontrees",
```

Comment:
Is the missing space between hashtag and account name of problem?

Reply:
Unified tags in all languages to `#StopTalkingStartPlanting #TrillionTrees @trilliontrees`